packages/fluttium_interfaces/lib/src/fluttium/fluttium.dart

```diff
@@ -0,0 +1,10 @@
+/// Barrel file for all the definitions required to parse the fluttium.yaml
+/// file.
+library fluttium;
```

Comment:
Should this be `fluttium_interfaces`? Also should the file path be `lib/src/fluttium_interfaces/fluttium_interfaces.dart`?

Reply:
While that is logical conclusion, my idea behind this was that this sub-library is for parsing the `fluttium.yaml`, therefore named `fluttium` and the other one (user_flow) is for parsing `<user_flow>.yaml` files.

Together their are the `fluttium_interfaces`

Reply:
Ah okay yeah I vaguely remember discussing this and that seems reasonable to me